packages/nocodb/src/models/DataReflection.ts

```
@@ -0,0 +1,177 @@
+      id?: string;
+      fk_workspace_id?: string;
+    },
+    ncMeta = Noco.ncMeta,
+  ) {
+    const condition = extractProps(params, ['id', 'fk_workspace_id']);
+
+    if (!condition.id && !condition.fk_workspace_id) {
+      throw NcError.badRequest('id or fk_workspace_id is required');
+    }
+
+    const dataReflection = await this.get(condition, ncMeta);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)